android/src/main/java/com/bolan9999/SpringScrollView.java

```diff
@@ -471,7 +471,7 @@ public void setContentOffset(float x, float y) {
         WritableMap contentOffsetMap = Arguments.createMap();
         contentOffsetMap.putDouble("x", PixelUtil.toDIPFromPixel(contentOffset.x));
         contentOffsetMap.putDouble("y", PixelUtil.toDIPFromPixel(contentOffset.y));
-        event.putMap("contentOffset", contentOffsetMap);
+        event.putMap("contentOffset", (ReadableMap)contentOffsetMap);
```

Comment:
Is this a safe cast? Won't throw an error on its own at some point?

Reply:
Yes, it should be a safe cast since WritableMap extends from ReadableMap. I'm not sure why it doesn't accept the WritableMap in the first place.